### PR TITLE
Add documentation for the Zeebe bindings

### PR DESCRIPTION
**Pull request by akkie**
## Description

Add documentation for the Zeebe bindings.

## Issue reference
https://github.com/dapr/components-contrib/issues/805

### CODE REVIEW

**Comment by orizohar** [comment]
@akkie - looks like this is a new component that is expected to be added in v1.2. Can you please rebase this PR to branch v1.2?

**Comment by akkie** [comment]
@orizohar done

File: daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md [hunk 1]
```diff
@@ -0,0 +1,583 @@
+﻿---
+type: docs
+title: "Zeebe command binding spec"
+linkTitle: "Zeebe command"
+description: "Detailed documentation on the Zeebe command binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/zeebe-commands/"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md]:
```suggestion
```

Note needed as this is a net-new page and not migrated from an existing location

File: daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+```
+
+## Spec metadata fields
+
+| Field                   | Required | Binding support |  Details | Example |
+|-------------------------|:--------:|------------|-----|---------|
+| gatewayAddr             | Y | Output | Zeebe gateway address                                                                     | `localhost:26500` | 
+| gatewayKeepAlive        | N | Output | Sets how often keep alive messages should be sent to the gateway. Defaults to 45 seconds  | `45s` | 
+| usePlainTextConnection  | N | Output | Whether to use a plain text connection or not                                             | `true,false` | 
+| caCertificatePath       | N | Output | The path to the CA cert                                                                    | `/path/to/ca-cert` |
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md:41]: How will this work on Kubernetes?

File: daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-jobworker.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+﻿---
+type: docs
+title: "Zeebe JobWorker binding spec"
+linkTitle: "Zeebe JobWorker"
+description: "Detailed documentation on the Zeebe JobWorker binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/zeebe-jobworker/"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-jobworker.md]:
```suggestion
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md (hunk 2 above)
Comment (akkie) [reply daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md:41]: You can mount a secret into the container with Kubernetes: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets

Comment (AaronCrawfis) [reply daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md:41]: Perfect, thanks!

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**